code/modules/mob/living/silicon/robot/robot_modules_vr.dm

```diff
@@ -147,3 +149,53 @@
 /obj/item/weapon/robot_module/robot/scrubpup/respawn_consumable(var/mob/living/silicon/robot/R, var/amount)
 	var/obj/item/device/lightreplacer/LR = locate() in src.modules
 	LR.Charge(R, amount)
+
+/obj/item/weapon/robot_module/robot/science
+	name = "Research Hound Module"
+	sprites = list(
+					"Research Hound" = "science",
+					)
+	channels = list("Science" = 1)
+	can_be_pushed = 0
+
+/obj/item/weapon/robot_module/robot/science/New(var/mob/living/silicon/robot/R)
+	src.modules += new /obj/item/weapon/dogborg/jaws/small(src)
+	src.modules += new /obj/item/device/dogborg/boop_module(src)
+	src.modules += new /obj/item/device/dogborg/tongue(src)
+	src.modules += new /obj/item/device/dogborg/sleeper/analyzer(src) //doesnt have a destructive analyzer in it yet
+	src.modules += new /obj/item/weapon/portable_destructive_analyzer(src)
+	src.modules += new /obj/item/weapon/gripper/research(src)
+//	src.modules += new /obj/item/weapon/gripper/no_use/organ/robotics(src)
+//	src.modules += new /obj/item/weapon/gripper/no_use/mech(src)
+//	src.modules += new /obj/item/weapon/gripper/no_use/loader(src)
+//	src.modules += new /obj/item/device/robotanalyzer(src)
+//	src.modules += new /obj/item/weapon/card/robot(src)
+//	src.modules += new /obj/item/weapon/weldingtool/electric/mounted/cyborg(src)
+	src.modules += new /obj/item/weapon/screwdriver/cyborg(src)
+	src.modules += new /obj/item/weapon/reagent_containers/glass/beaker/large(src)
+	src.modules += new /obj/item/weapon/storage/part_replacer(src)
+//	src.modules += new /obj/item/weapon/shockpaddles/robot/hound/jumper(src)
+//	src.modules += new /obj/item/weapon/melee/baton/slime/robot(src)
+//	src.modules += new /obj/item/weapon/gun/energy/taser/xeno/robot(src)
+	src.emag = new /obj/item/weapon/hand_tele(src)
+	R.icon 		 = 'icons/mob/widerobot_vr.dmi'
+	R.hands.icon = 'icons/mob/screen1_robot_vr.dmi'
+	//R.icon_state = "science"
+	R.pixel_x 	 = -16
+	R.old_x 	 = -16
+	..()
+
+	var/datum/matter_synth/nanite = new /datum/matter_synth/nanite(10000)
+	synths += nanite
+	var/datum/matter_synth/wire = new /datum/matter_synth/wire()						//Added to allow repairs, would rather add cable now than be asked to add it later,
+	synths += wire																		//Cable code, taken from engiborg,
+
+	var/obj/item/stack/nanopaste/N = new /obj/item/stack/nanopaste(src)
+	N.uses_charge = 1
+	N.charge_costs = list(1000)
+	N.synths = list(nanite)
+	src.modules += N
```

Comment:
The infinite wire, nanopaste, and cable coils should be removed as well. Wouldn't be needed by a normal scientist borg.

Reply:
What of robotics repair? Although I guess this may cause for a self repairing borg.

Reply:
Well even if it did still have the card i dont know if it could open itself

Reply:
@SpadesNeil Per our discussion in the admin chat, we decided we didn't want it doing robotics, hence asking for the removal.